.travis.yml

```diff
@@ -3,49 +3,54 @@ branches:
     - main
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
-language: rust
-rust:
-  - stable
+os: linux
+dist: focal
```

Comment:
Since I was touching this file anyway I explicitly set the version here and picked the most recent ubuntu release.